scripts/popup.js

```diff
@@ -221,20 +221,10 @@ function show_news() {
     var to_check_url = url.replace(/^https?:\/\/(www\.)?/, '')
     var news_host = to_check_url.split('.')[0]
```

Comment:
This URL string processing with regex replace and splitting seems really error prone.
Why don't you use `URL()` to parse the URL properly and get `url.hostname`? That would be certainly better.